src/main/java/seedu/address/logic/parser/appointments/SettleAppCommandParser.java

```diff
@@ -52,7 +52,7 @@ public ReversibleActionPairCommand parse(String args) throws ParseException {
         try {
             if (lastShownList.size() == 0) {
                 throw new ParseException(Messages.MESSAGE_NOTHING_SETTLE + "\n"
-                        + "No need: " + "settleappt " + args);
+                        + "No need: " + "settleappt" + args);
```

Comment:
Change this to just "throw new ParseException(MESSAGE_NOTHING_SETTLE);"

Reply:
solved in other pr